src/Pages/logIn.cshtml

```diff
@@ -1,25 +1,25 @@
 ﻿@page
 @model ePiggyWeb.Pages.logInModel
-@{
-}
 
 <div class="container login-container">
     <div class="login-form-2">
         <h3>Sign In</h3>
-        <form>
+        <form method="post">
             <div class="form-group">
-                <input type="text" class="form-control" placeholder="Your Email *" value="" />
+                <input asp-for="Email" type="text" class="form-control" placeholder="Your Email *" value="" />
             </div>
             <div class="form-group">
-                <input type="password" class="form-control" placeholder="Your Password *" value="" />
+                <input asp-for="Password" type="password" class="form-control" placeholder="Your Password *" value="" />
             </div>
             <div class="form-group">
                 <input type="submit" class="btnSubmit" value="Login" />
             </div>
             <div class="form-group">
-
                 <a href="#" class="ForgetPwd" value="Login">Forget Password?</a>
             </div>
+            <div class="form-group">
+                <a asp-page="/Register" class="ForgetPwd" value="Login">New User?</a>
+            </div>
```

Comment:
Just curious, how does this work? I know there are some issues at the moment but in general at least

Reply:
This is a basic html form, which allows to use `Post` method when some button (in this case Login button) is pressed.
interesting part is `asp-for`. It allows to save data from input fields into Properties that are inside `logIn.cshtml.cs` and then use them or pass them. It is interesting because it somewhat different on how Javascript handles input fields using node.js, but at the same time somewhat similar, but that's not important. 
If you have any further questions don't hesitate to ask :)

Reply:
Thanks for the explanation! The asp-for is actually what got me to ask the question in the first place